proposals/0002-Turbo-Modules.md

```diff
@@ -0,0 +1,53 @@
+# RFC0002: Turbo Modules ™ 
+_Its blazing fast_
+
+## Summary
+
+React Native uses [Native Modules](https://facebook.github.io/react-native/docs/native-modules-ios) as a way for JavaScript  to invoke functions in Java/ObjC. Listed under the API section in the [React Native documentation](https://facebook.github.io/react-native/docs/getting-started), popular native modules include Geolocation, Device Information, Async Storage, etc.
```

Comment:
I would explicitly mention here that Native Modules does *not* directly tight with UI rendering pipeline and that currently-existing View Managers (which are also Native Modules) will go away soon.

Reply:
@shergin Is there more information on the coming removal of View Managers?

Reply:
view managers will be handled as part of fabric rollout: https://github.com/react-native-community/discussions-and-proposals/issues/4

exact details for migration path is TBD